.laminas-ci.json

```diff
@@ -3,30 +3,6 @@
     "ini": ["apc.enabled=1", "apc.enable_cli=1"],
     "backwardCompatibilityCheck": true,
     "additional_checks": [
-        {
-            "name": "Integration tests with memory adapter",
-            "job": {
-                "command": "composer require laminas/laminas-cache-storage-adapter-memory && ./vendor/bin/phpunit --testsuite 'Laminas Cache Integration Tests'",
-                "php": "*",
-                "dependencies": "lowest"
-            }
-        },
-        {
-            "name": "Integration tests with memory adapter",
-            "job": {
-                "command": "composer require laminas/laminas-cache-storage-adapter-memory && ./vendor/bin/phpunit --testsuite 'Laminas Cache Integration Tests'",
-                "php": "*",
-                "dependencies": "latest"
-            }
-        },
-        {
-            "name": "Integration tests with memory adapter",
-            "job": {
-                "command": "composer require laminas/laminas-cache-storage-adapter-memory && ./vendor/bin/phpunit --testsuite 'Laminas Cache Integration Tests'",
-                "php": "@lowest",
-                "dependencies": "locked"
-            }
-        },
```

Comment:
I missed the fact that the tests were already running with APCu.

So we're **removing** the integration tests with the `memory` adapter. This seems a bit off to me, because our own adapters _really_ should be passing with these tests.

I note that the tests are currently passing in the memory adapter repository.

I'll merge for now, but it would be good to know why the memory adapter is failing here

Reply:
see https://github.com/laminas/laminas-cache-storage-adapter-test/pull/72 :)